packages/astroport/src/factory.rs

```diff
@@ -24,112 +36,167 @@ impl Display for PairType {
 }
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+/// ## Description
+/// This structure describes a configuration of pair.
 pub struct PairConfig {
+    /// Sets pair contract code ID which are allowed to create pair
     pub code_id: u64,
+    /// Sets the type of pair available in [`PairType`]
     pub pair_type: PairType,
+    /// Sets a pair total fees bps
     pub total_fee_bps: u16,
+    /// Sets a pair fees bps
     pub maker_fee_bps: u16,
+    /// Used to check if pair configuration is disabled
```

Comment:
We disable pair configs instead of removing them. If it is disabled, new pairs cannot be created, but existing ones can still obtain proper settings, such as fee amounts